rubygem-fog/rubygem-fog.spec

```diff
@@ -49,8 +49,7 @@ Requires: %{?scl_prefix}rubygem(fog-xml) >= 0.1.1
 Requires: %{?scl_prefix}rubygem(fog-xml) < 0.2.0
 Requires: %{?scl_prefix}rubygem(ipaddress) >= 0.5
 Requires: %{?scl_prefix}rubygem(ipaddress) < 1.0
-Requires: %{?scl_prefix_ruby}rubygem(json) >= 1.8
-Requires: %{?scl_prefix_ruby}rubygem(json) < 2.0
+Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.0
```

Comment:
This requires us to bump to the newer ruby SCL. Will require some coordination with @ehelms.